src/components/Footer.tsx

```diff
@@ -59,7 +57,7 @@ export default function () {
           Privacy policy
         </FooterLink>
       </div>
-      {!md && (
+      <span className={displayToMd}>
```

Comment:
`div` inside `span` is not the best way, let's use `div` here too, btw, you can include this `displayToMd` into `socialContainer` styles and avoid double-nesting

Reply:
Did it, thx